src/compute-pr-actions.ts

```diff
@@ -133,6 +134,7 @@ export interface ExtendedPrInfo extends PrInfo {
     readonly isAuthor: (user: string) => boolean; // specialized version of sameUser
 }
 function extendPrInfo(info: PrInfo): ExtendedPrInfo {
+    const headCommitAbbrOid = abbrOid(info.headCommitOid);

```

Comment:
Nice :+1: Given the `abbrOid()` helper, can we remove `headCommitAbbrOid` from `ExtendedPrInfo` and replace it with `abbrOid(headCommitOid)` à la #275?

Reply:
I first disliked it since adding the function call in all places is more verbose, but then I realized that they're all in `process`...

See #308.